src/components/Pages/Home.jsx

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+
+export default function Home() {
+  return (
+    <div
```

Comment:
should use style components consistently instead of in-line style code; Robert: never do in-line styling unless u use bootstrap

Reply:
done. added styling into index.css